e-store

barsukovmd opened the pull request:
не запускается магазин, пишет следующее...

barsukovmd committed:
Spring mvc estore testing

barsukovmd committed:
Spring mvc estore testing

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring update

barsukovmd committed:
spring boot test

barsukovmd committed:
spring boot test

barsukovmd committed:
spring boot test

barsukovmd committed:
spring boot test

barsukovmd commented:
<img width="1122" alt="Screenshot 2023-04-30 at 5 57 25 PM" src="https://user-images.githubusercontent.com/116370320/235377980-6f0b02b5-2d4d-4fb2-981f-83d3d2636485.png">

tmspavel commented:
> <img alt="Screenshot 2023-04-30 at 5 57 25 PM" width="1122" src="https://user-images.githubusercontent.com/116370320/235377980-6f0b02b5-2d4d-4fb2-981f-83d3d2636485.png">

а в базе данных как таблица называется?

barsukovmd committed:
spring boot test

barsukovmd committed:
maven dependency to fast CPU after intellij IDEA update 2023

<dependency>
<groupId>com.github.sviperll</groupId>
<artifactId>maven-profiledep-extension</artifactId>
<version>0.24-beta2</version>
</dependency>

barsukovmd committed:
spring boot test

tmspavel commented on Lesson40SpringMvcTest/src/main/java/us/config/SpringConfig.java:
```diff
@@ -0,0 +1,50 @@
+package us.config;
```
com.tms лучше использовать а не us
ну или com.eshop

tmspavel reviewed (commented):
посмотрел!
